package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node node_modules/gulp/bin/gulp.js --color"
+    "start": "set NODE_ENV=dev && node node_modules/gulp/bin/gulp.js --color",
+    "deploy": "set NODE_ENV=prod && node node_modules/gulp/bin/gulp.js deploy --color"
```

Comment:
You do not have to set/export. The proper way of using NODE_ENV is `NODE_ENV={env} node app.js`